config/ember-try.js

```diff
@@ -1,129 +1,41 @@
+/* eslint-env node */
```

Comment:
This file is not the same structure as being used in all other related ember cli upgrade PRs